test/e2e_node/cgroup_manager_test.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
One more thing is that, running the test in pod makes the test portable across cluster and node e2es.
WDYT @Random-Liu ?